tests/regression/detection/test_detection.py

```diff
@@ -0,0 +1,263 @@
+"""Tests for Detection with OTX CLI"""
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+import json
+import os
+from pathlib import Path
+from timeit import default_timer as timer
+
+import pytest
+
+from otx.cli.registry import Registry
+from tests.regression.regression_test_helpers import (
+    REGRESSION_TEST_EPOCHS,
+    TIME_LOG,
+    get_result_dict,
+    load_regression_configuration,
+)
+from tests.test_suite.e2e_test_system import e2e_pytest_component
+from tests.test_suite.run_test_command import (
+    get_template_dir,
+    nncf_eval_testing,
+    nncf_optimize_testing,
+    otx_deploy_openvino_testing,
+    otx_eval_compare,
+    otx_eval_deployment_testing,
+    otx_eval_openvino_testing,
+    otx_export_testing,
+    otx_train_testing,
+    pot_eval_testing,
+    pot_optimize_testing,
+)
+
+# Configurations for regression test.
+TASK_TYPE = "detection"
+TRAIN_TYPE = "supervised"
+LABEL_TYPE = "multi_class"
+
+otx_dir = os.getcwd()
+templates = Registry("otx/algorithms/detection").filter(task_type=TASK_TYPE.upper()).templates
+templates_ids = [template.model_template_id for template in templates]
+
+result_dict = get_result_dict(TASK_TYPE)
+result_dir = f"/tmp/regression_test_results/{TASK_TYPE}"
+Path(result_dir).mkdir(parents=True, exist_ok=True)
+
+# Detection
+detection_regression_config = load_regression_configuration(otx_dir, TASK_TYPE, TRAIN_TYPE, LABEL_TYPE)
+detection_data_args = detection_regression_config["data_path"]
+detection_data_args["train_params"] = ["params", "--learning_parameters.num_iters", REGRESSION_TEST_EPOCHS]
+
+
+class TestRegressionDetection:
+    def setup_method(self):
+        self.label_type = LABEL_TYPE
```

Comment:
For the classification case, we need to use `self.label_type` since other test classes have different label_type.
However, for the detection case, we can avoid the `self.label_type`, I'm going to reflect it

i.e. 
``` python3
class TestRegressionMultiClassClassification:
    def setup_method(self):
        self.label_type = "multi_class"
        self.performance = {}

class TestRegressionMultiLabelClassification:
    def setup_method(self):
        self.label_type = "multi_label"
        self.performance = {}
```